frontend/resources/translations/vi.json

```diff
@@ -0,0 +1,72 @@
+{
+  "appName": "AppFlowy",
+  "defaultUsername": "Tôi",
+  "welcomeText": "Chào mừng bạn đến với @:appName",
+  "welcomeTo": "Chào mừng bạn đến",
+  "githubStarText": "Sao trên GitHub",
+  "subscribeNewsletterText": "Đăng ký tin",
+  "letsGoButtonText": "Bắt đầu nhanh",
+  "title": "Tiêu đề",
+  "youCanAlso": "Bạn cũng có thể",
+  "and": "Và",
+  "failedToOpenUrl": "Không mở được url: {}",
+  "blockActions": {
+    "addBelowTooltip": "Nhấn vào đây để thêm vào bên dưới",
+    "addAboveCmd": "Alt+nhấp chuột",
+    "addAboveMacCmd": "Tùy chọn+nhấp chuột",
+    "addAboveTooltip": "để thêm ở trên",
+    "dragTooltip": "Kéo để di chuyển",
+    "openMenuTooltip": "Bấm để mở menu"
+  },
```

Comment:
Hey there, thanks for the contribution. What is the difference between vi.json and vi-VN? if there are differences in vi.json from vi-VN, please specify the country code for vi.json.